core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -1069,6 +1079,7 @@ class ZKCheckedEphemeral(path: String,
     val prefix = path.substring(0, index)
     val suffix = path.substring(index, path.length)
     debug("Path: %s, Prefix: %s, Suffix: %s".format(path, prefix, suffix))
+    info("Is secure: " + isSecure)
```

Comment:
This is information that is good for the operator to know, but I agree that the message is obscure. I can do one of two things here:

1- Remove the comment altogether if you think this isn't the way info messages are being used across the project
2- Improve the text of the message to add more detail

I'm fine either way.